src/core/variable.jl

```diff
@@ -119,8 +119,8 @@ function variable_mc_branch_power_real(pm::_PM.AbstractPowerModel; nw::Int=pm.cn
     if bounded
         for (l,i,j) in ref(pm, nw, :arcs)
             smax = _calc_branch_power_max(ref(pm, nw, :branch, l), ref(pm, nw, :bus, i))
-            set_upper_bound.(p[(l,i,j)],  smax)
-            set_lower_bound.(p[(l,i,j)], -smax)
+            set_upper_bound.(p[(l,i,j)],  1000000) #hacked bounds as _calc_branch_power_max involves multiplication of 0 with inf which is undf
+            set_lower_bound.(p[(l,i,j)], -1000000)
```

Comment:
I think the `undf` should be handled inside of the `set_lower_bound` and `set_upper_bound` function (which are located in `/src/core/data.jl`), instead of adding a hack here, which could cause unintended problems.